core/app/models/spree/tax/output/item.rb

```diff
@@ -0,0 +1,10 @@
+module Spree
+  module Tax
+    module Output
```

Comment:
_Maybe_ we can come up with a more obvious name to what these classes represent than `Output`. Maybe `Spree::Tax::TaxedItem`? Naming is hard.